contents/docs/data/persons.mdx

```diff
@@ -55,3 +55,65 @@ Clicking on a user opens their profile and shows all their properties.
 ## Deleting person data with the API
 
 You can also delete persons data via the API. See the [Data Deletion docs](/docs/privacy/data-deletion) for more information.
+
+## Events with person processing
+
+Events with person processing create or update an associated person profile (see above). This allows for the following features:
```

Comment:
What is "person processing" or "events with person processing"? I read this section and I have no idea what we're referring to here